QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/BasePage.java

```diff
@@ -67,4 +71,9 @@ public static String getRandomBoingEmail() {
         }
         return salt + "@boing.rs";
     }
+
+    public static void explicitWait(WebElement element, WebDriver driver) {
```

Comment:
this method should ne renamed. Instead of explicit wait it should point to method purpose(waitForElementVisible or waitForElementVisibility)